storybook/stories/_support/utils.js

```diff
@@ -120,6 +120,7 @@ export const generateWallet = (
     amount: new BigNumber(amount).dividedBy(LOVELACES_PER_ADA),
     availableAmount: new BigNumber(amount).dividedBy(LOVELACES_PER_ADA),
     reward: new BigNumber(reward).dividedBy(LOVELACES_PER_ADA),
+    // $FlowFixMe
```

Comment:
Please remove this and fix flow issues, this is probably because you added `?`WalletAssets